.eslintrc

```diff
@@ -8,12 +8,14 @@
   "extends": ["airbnb", "prettier"],
   "plugins": ["prettier"],
   "rules": {
-    "prettier/prettier": ["error"],
+    // "prettier/prettier": ["error"],
     "import/no-extraneous-dependencies": 0,
     "react/jsx-filename-extension": 0,
     //  proptypes is not required : 
     "react/prop-types": 0,
     "no-unused-vars":0,
-    "react/jsx-props-no-spreading":0
+    "react/jsx-props-no-spreading":0,
+    "endOfLine": 0,
+    "react/no-array-index-key": 0
   }
 }
```

Comment:
as we build in  a big team, please dont remove linters/or prettier configs ( to check betters diffs and kind of)